src/style.css

```diff
@@ -0,0 +1,137 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --background-color-blue: #00acdc;
+  --background-color-white: #fff;
+  --dark-color-text: #060b35;
+  font-family: "Roboto", sans-serif;
+}
+
+h6,
+p {
+  margin: 0;
+}
+
+.card {
```

Comment:
try to add around padding for card, instead of adding side padding for each element. Should be smth like this
![image](https://github.com/mate-academy/layout_product-cards/assets/54936044/fba71103-3d79-4d83-8e10-3e339e5c5098)